packages/language-server/src/__test__/__fixtures__/multi-file/config-with-broken-import/prisma.config.ts

```diff
@@ -0,0 +1,7 @@
+import 'whatever'
+
+export default {
+  datasource: {
+    url: 'dummy',
+  },
+}
```

Comment:
Just to be extra-safe and future-proof, let's have another fixture referencing a non-existing variable:

```ts
import { env } from 'prisma/config'

export default {
  datasource: {
    url: env('UNSET_ENV_VAR'),
  },
}
```